### PR TITLE
#162042477 Improve the user interface design

### PR DESCRIPTION
**Pull request by AmosWels**
#### What does this PR do?
- Improves the User Interface of the application
#### Description of Task to be completed?
- Modifies the navigation bar
- Modifies the create article page
- Modifies the view one article page
#### How should this be manually tested?
- Visit the application at [ah haven](https://ah-frontend-lannister-pr-20.herokuapp.com/) 
#### What are the relevant pivotal tracker stories?
N/A
##### Create Article Page
image

#### VIew all articles
image

#### View Article
image

#### What are the relevant pivotal tracker stories?
[#162042477](https://www.pivotaltracker.com/story/show/162042477)

### CODE REVIEW

**Comment by LehruAngela** [comment]
@AmosWels rebase with develop

**Comment by AmosWels** [comment]
> @AmosWels rebase with develop

>I have rebased but finishing with the changes

File: src/assets/articleAssets/rating.scss [hunk 1]
```diff
@@ -32,3 +32,26 @@
...
+    padding: 5px 0;
+
+    /* Position the tooltip */
+    position: absolute;
+    z-index: 1;
+}
+
+.tooltip:hover .tooltiptext {
+    visibility: visible;
+}
```
Comment (LehruAngela) [inline src/assets/articleAssets/rating.scss]: Add new line at end of file

**Review by LehruAngela — CHANGES_REQUESTED** [review]

File: src/assets/articleAssets/rating.scss (hunk 1 above)
Comment (AmosWels) [reply src/assets/articleAssets/rating.scss]: I have added the new line

**Pull request merged**